### DISCUSSION TITLE
MAX_PRICE for eu shops doesn' work

### DISCUSSION THREAD

**Discussion by ManicPumpkin** [jef/streetmerchant#2906] [Help]
Hello, 

I'm currently trying to figure out, how to set the max price for specific series. Currently my setting looks like this:

```
MAX_PRICE_SERIES_3080=1200
MAX_PRICE_SERIES_3080TI=1500
```

If I start the streetmerchant, then it finds the GPU on Amazon.de. But the maximum price does not match. For example, I've set the 3080 to 1.200€, then the merchant finds the GPU for >2.000€. Could it be possible that the ID tag (#priceblock_ourprice) for the HTML isn't correct? Or did I set it up incorrectly?

Thank you very much!

Best regards,
MAN!C.

**Comment by Mr-Tech-13** [comment]
https://github.com/jef/streetmerchant/pull/2856 this may help you
